docs/stock/cumulative-sum.md

```diff
@@ -6,4 +6,4 @@ The Cumulative Sum tool sums (cumulates) all the previous values with the curren
 For the following data: `[10, 8, 15, 20, 8, 15]` the Cumulative Sum returns `[10, 18, 33, 53, 61, 76]` (see the graph below):
 ![cumulative-sum.png](cumulative-sum.png)
 
-The `cumulative` can be enabled in the chart's options using the [series.cumulative](https://api.highcharts.com/highstock/plotOptions.series.cumulative) property or enabled/disabled by the [series.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Series#setCumulative) method or on all the series belonging to a specific y-axis by the [axis.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Axis#setCompare) method.
\ No newline at end of file
+The `cumulative` can be enabled in the chart's options using the [series.cumulative](https://api.highcharts.com/highstock/plotOptions.series.cumulative) property or enabled/disabled by the [series.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Series#setCumulative) method or on all the series belonging to a specific y-axis by the [yAxis.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Axis#setCompare) method.
```

Comment:
`[yAxis.setCumulative()](https://api.highcharts.com/class-reference/Highcharts.Axis#setCompare)`

Here the `setCumulative` leads to the `setCompare`